src/lib/operators/jit_operator/operators/jit_aggregate.cpp

```diff
@@ -21,6 +21,18 @@ std::string JitAggregate::description() const {
   return desc.str();
 }
 
+void JitAggregate::before_specialization(const Table& in_table, std::vector<bool>& tuple_nullable_information) {
+  // Update the groupby and aggregate column nullable information from the tuple_nullable_information
+  for (auto& groupby_column : _groupby_columns) {
+    groupby_column.tuple_entry.is_nullable = tuple_nullable_information[groupby_column.tuple_entry.tuple_index];
+    groupby_column.hashmap_entry.is_nullable = tuple_nullable_information[groupby_column.tuple_entry.tuple_index];
+  }
+
+  for (auto& aggregate_column : _aggregate_columns) {
+    aggregate_column.tuple_entry.is_nullable = tuple_nullable_information[aggregate_column.tuple_entry.tuple_index];
```

Comment:
Wouldn’t this have to be updated? For example, `MIN(a)` could return NULL if `a` is a `NOT NULL` but empty column

Reply:
I'd add a comment saying explaining this.

Reply:
Added a comment.